tern/utils/general.py

```diff
@@ -77,7 +77,11 @@ def get_git_rev():
     '''Assuming we are operating within a git repository, get the SHA
     of the current commit'''
     command = ['git', 'show', '--format=%H', 'HEAD']
-    output = subprocess.check_output(command)  # nosec
+    try:
+        output = subprocess.check_output(command)  # nosec
+    except subprocess.CalledProcessError as e:
+        # FIXME
+        output = "HEAD"
```

Comment:
Hmm...we might want to do away with the git sha1 reporting of the version. Would it be possible to read the version from setup.py for example?

Reply:
Yep, another option would be to have a version file within the module and load it within setup.py and at runtime.

Reply:
I like that idea better!